tests/clean/basic/test.sh

```diff
@@ -14,6 +15,13 @@ rlJournalStart
         done
     rlPhaseEnd
 
+    rlPhaseStartTest "Create a couple of runs in non-default root workdir"
+        for id in {001..003}; do
+            rlRun "tmt --feeling-safe run --id clean-$id --workdir_root $test_root"
```

Comment:
Afaik, we want to have feeling safe handled by an env var.

Reply:
Updated:)